minifi-bootstrap/src/test/java/org/apache/nifi/minifi/bootstrap/configuration/notifiers/util/TestRestChangeNotifierCommon.java

```diff
@@ -84,6 +78,14 @@ public void testFileUpload() throws Exception {
 
         assertEquals("The result of notifying listeners:\nMockChangeListener successfully handled the configuration change\n", response.body().string());
 
-        assertEquals(new String(Files.readAllBytes(file.toPath())), mockChangeListener.getConfFile());
+        // Ignoring the system dependent line ending.
+        String confFile = mockChangeListener.getConfFile();
+        if (confFile.endsWith("\n") || confFile.endsWith("\r")) {
```

Comment:
Do you think it would be better to just use a StringUtils#trim on the returned conf file?